charts/keycloak/values.yaml

```diff
@@ -22,3 +22,54 @@ keycloak:
     username: keycloak
     database: keycloak
     passwordVaultKey: secret/data/global/keycloak
+
+  # Toggle to enable/disable default realm configuration
+  defaultConfig: false
+  # Default realm configuration
+  defaultRealm:
+    realm: "ztvp"
+    displayName: "ZTVP Realm"
+    enabled: true
+    registrationAllowed: false
+    clients:
+      - clientId: "quarkus-app"
+        name: "quarkus"
+        publicClient: true
+        redirectUris:
+          - "https://cat-server.example.com/*"
```

Comment:
For now, we can specify `*`. In the longterm, we will want to lock it down. Something to think about is how we will eventually inject the URL at runtime